### ISSUE TITLE
Train and validation accuracy become always 0.00000

### ISSUE THREAD

**Issue by k-hashimoto** [apache/incubator-mxnet#4977]
Hi,

I tried to fit own images with simple CNN today(I was git cloned latest version of MXNet). But accuracy of train and validation data are always zero. 

INFO:root:Epoch[0] Batch [10]	Speed: 30.93 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [20]	Speed: 28.97 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [30]	Speed: 28.99 samples/sec	Train-accuracy=0.000000
........

I think this issue come from my misunderstandings about MXNet. I can only know "Spped" and "Train-accuracy" during training. But I want a lot of information to debug this. How can I get more detailed information during training?

A part of code of input data is
`
val_data_iter = mx.io.ImageRecordIter(path_imgrec='imgs/validation_DDD.rec',
data_shape=data_shape,
batch_size=batch_size,
shuffle=False,
rand_crop=False
)

train_data_iter = mx.io.ImageRecordIter(path_imgrec='imgs/train_DDD.rec',
data_shape=data_shape,
batch_size=batch_size,
shuffle=False,
rand_crop=False
)
'

My CNN is ...

data = mx.symbol.Variable('data')
conv1 = mx.sym.Convolution(data=data, kernel=(3,3), num_filter=32)
act1 = mx.sym.Activation(data=conv1, act_type="relu")
pool1 = mx.sym.Pooling(data=act1, pool_type="max", kernel=(2,2), stride=(2,2))
conv2 = mx.sym.Convolution(data=pool1, kernel=(3,3), num_filter=64)
act2 = mx.sym.Activation(data=conv2, act_type="relu")
pool2 = mx.sym.Pooling(data=act2, pool_type="max", kernel=(2,2), stride=(2,2))
flatten = mx.sym.Flatten(data=pool2)
fc1 = mx.symbol.FullyConnected(data=flatten, num_hidden=512)
act3 = mx.sym.Activation(data=fc1, act_type="relu")
fc2 = mx.sym.FullyConnected(data=act3, num_hidden=4)
net = mx.sym.SoftmaxOutput(data=fc2, name='softmax')

Now do fitting....

mod = mx.mod.Module(net)
mod.fit(train_data_iter,
eval_data=val_data_iter,
optimizer_params={'learning_rate':0.01, 'momentum': 0.9},
num_epoch=n_epoch,
epoch_end_callback=checkpoint,
batch_end_callback=batch_end_callback
)

**Comment by k-hashimoto** [comment]
Hi,

I try to fit own images with simple 3 layers CNN today(I was git clone latest version of MXNet). But accuracy of train and validation data are always zero. 

INFO:root:Epoch[0] Batch [10]	Speed: 30.93 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [20]	Speed: 28.97 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [30]	Speed: 28.99 samples/sec	Train-accuracy=0.000000

"""
"""

**Comment by k-hashimoto** [comment]
I found my mistake. ...

**Comment by BrettLL** [comment]
what is your mistake?? I met the same problem!!

**Comment by yajiedesign** [comment]
This issue is closed due to lack of activity in the last 90 days. Feel free to reopen if this is still an active issue. Thanks!
